fullscreen/api/element-request-fullscreen-dialog.html

```diff
@@ -2,15 +2,22 @@
 <title>Element#requestFullscreen() for dialog element</title>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-<script src="../trusted-click.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
 <div id="log"></div>
 <dialog></dialog>
 <script>
 async_test(t => {
   var dialog = document.querySelector("dialog");
   assert_true(dialog instanceof HTMLDialogElement);
-  trusted_request(t, dialog);
+
+  document.body.addEventListener("click", () => {
```

Comment:
You shouldn't need to leave this in when you're using `test_driver.bless`. I guess it was for testing?

Reply:
Yeah, I forgot to remove it